.github/workflows/wss.yml

```
@@ -0,0 +1,21 @@
+name: Whitesource
+
+on:
+  release:
+    types: [published]
+
+env:
+  PARAM_REPO: xapi-service
+  PARAM_SERVICE: xapiserver
```

Do we want to keep the naming consistent here @wobrien @AodhanLP?
```suggestion
  PARAM_SERVICE: xapiservice
```

---

Hey @ryansmith94 ,

The service parameter has already been declared in a separate [WhiteSource config file](https://github.com/HT2-Labs/whitesource-configs) (I don't have access to this file) so I cannot change the service name, maybe Willie or @christopher-learningpool can help advise on this. Thanks.

---

Hey @ryansmith94 ,

Myself and Chris have updated the parameter service name to be xapiservice to maintain consistentcy here. Thanks.